doc/classes/Tree.xml

```diff
@@ -341,6 +341,9 @@
 		<member name="allow_search" type="bool" setter="set_allow_search" getter="get_allow_search" default="true">
 			If [code]true[/code], allows navigating the [Tree] with letter keys through incremental search.
 		</member>
+		<member name="auto_tooltip" type="bool" setter="set_auto_tooltip" getter="is_auto_tooltip_enabled" default="true">
+			If [code]true[/code], tree items with no tooltip assigned display their title as their tooltip.
```

Comment:
TreeItems don't have "title".

Reply:
Already noticed that one and fixed it, must have gotten reverted at some point, have fixed again